cou/utils/openstack.py

```diff
@@ -66,7 +67,7 @@ class OpenStackCodenameLookup:
     """Class to determine compatible OpenStack codenames for a given component."""
 
     _OPENSTACK_LOOKUP: OrderedDict = OrderedDict()
-    _DEFAULT_CSV_FILE = os.path.join(os.path.dirname(__file__), "openstack_lookup.csv")
+    _DEFAULT_CSV_FILE = os.path.join(Path(__file__).parent, "openstack_lookup.csv")
```

Comment:
You do not need to use `os.path.join` for `Path` object
```suggestion
    _DEFAULT_CSV_FILE = Path(__file__).parent / "openstack_lookup.csv"
```

Reply:
I did not know that Path implements `__truediv__` . Thanks

Reply:
done